README.md

```diff
@@ -1,3 +1,33 @@
 # Strapi application
 
-A quick description of your strapi application
+Setting up Strapi
+
+The first thing you do after forking the ACC-repo is to run "npm run build" and "npm run develop" to start the server, but as you will see, its not that easy.
```

Comment:
also npm install. Can you make this more of a bullet point list or a step by step?